cw_bitcoin/lib/litecoin_wallet.dart

```diff
@@ -880,7 +883,29 @@ abstract class LitecoinWalletBase extends ElectrumWallet with Store {
       ));
       final tx2 = BtcTransaction.fromRaw(hex.encode(resp.rawTx));
 
+      // check if the transaction doesn't contain any mweb inputs or outputs:
+      final transactionCredentials = credentials as BitcoinTransactionCredentials;
+
+      bool hasMwebInput = false;
+      bool hasMwebOutput = false;
+
+      for (final output in transactionCredentials.outputs) {
+        if (output.extractedAddress?.toLowerCase().contains("mweb") ?? false) {
+          hasMwebOutput = true;
+          break;
+        }
+      }
+
+      if (tx2.mwebBytes != null && tx2.mwebBytes!.isNotEmpty) {
+        hasMwebInput = true;
+      }
+
+      if (!hasMwebInput && !hasMwebOutput) {
+        return tx;
+      }
```

Comment:
**Enhancement:**
would have been better to do it right before/after the `if (!mwebEnabled)` check, so we don't await some functions that we don't need

also what about the TODO you added regarding type casting BitcoinScriptOutput?
it should work, otherwise the compiler would have flagged it as non supported type